static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
1. What prefix is that? doesn't the strings present as they should?
2. Correct me if I'm wrong - process.py fills the DB. where should the localization be done if so?
3. load_extra_data pulls many other properties, would you like each option to have a separate field?